doc/neo-tree.txt

```diff
@@ -128,6 +128,12 @@ want to change the current working directory. If you include this flag, it will
 automatically change the directory without prompting. This option implies
 "reveal", so you do not need to specify both.
 
+selector~
+This is a boolean flag. When you specifically set this to false (|selector=false|)
```

Comment:
The `|word|` syntax is for creating a link to a tag. Tags are created from words in this format:`*word*`. There is no `*selector=false*` tag so this is just a dead link. You probably want to use backticks here instead:

```suggestion
This is a boolean flag. When you specifically set this to false (`selector=false`)
```

Reply:
Thanks. As you might have guessed, I'm very new to vim docs lol.
Fixed ;)